onprc_ehr/resources/queries/study/ClinpathRuns.query.xml

```diff
@@ -129,6 +129,15 @@
                     <column columnName="condition">
                         <columnTitle>Condition</columnTitle>
                     </column>
+                    <column columnName="vet">
+                        <columnTitle>Vetrinarian</columnTitle>
+                        <fk>
+                            <fkDbSchema>onprc_ehr</fkDbSchema>
+                            <fkTable>Labwork_Requestor_Vets</fkTable>
+                            <fkColumnName>userid</fkColumnName>
+                            <fkDisplayColumnName useRawValue="true"/>
```

Comment:
Do you really want to show the userid value (an integer) instead of a user name? The useRawValue="true" forces it to use the underlying key value.